src/HTTP/Response/rendering/ResponseRenderingStrategy.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace ILIAS\HTTP\Response\rendering;
+
+use Psr\Http\Message\ResponseInterface;
+
+/**
+ * Interface ResponseRenderingStrategy
+ *
+ * @author  Nicolas Schäfli <ns@studer-raimann.ch>
+ */
+interface ResponseRenderingStrategy {
```

Comment:
The name of this interface is wrong. As the comment for `renderResponse` and the implementation for `DefaultResponseRenderingStrategy` show, this is a `ResponseSenderStrategy`. It certainly does more then rendering.

Why is this a `Strategy` btw? How is this different from a `ResponseSender`?